magma/bits.py

```diff
@@ -78,6 +80,7 @@ def __str__(cls):
 
 class Bits(Array, AbstractBitVector, metaclass=BitsMeta):
     __hash__ = Array.__hash__
+    hwtypes_T = ht.BitVector
```

Comment:
are these used anywhere besides locally? if outside users don't need them, would suggest not including them in the class namespace.

Reply:
I think leaving it in the class namespace could be useful for inheritance.  For example, if a subclass a UInt, but I want to inherit the simulate logic (e.g. just a named type that behaves exactly as a Uint) it would be convenient to just inherit this property.

Reply:
Similarly, if the user wanted to override the simulation implementation of the primitive, they could override this name and then reuse the primitive simulation methods (they just need to define the operators on a new type with the desired behavior)

Reply:
(that's what UInt and SInt are doing by simply overriding this name, while inheriting the Bits logic for the primitive simulation methods)